clang/test/3C/unwritable_rootcauses.h

```diff
@@ -0,0 +1,15 @@
+
+   
+void* ptr; // unwritable-expected-warning {{2 unchecked pointers: Declaration in non-writable file}}
+
+
+int arr[] = {1,2,3,4};// unwritable-expected-warning {{1 unchecked pointer: Declaration in non-writable file}}  // unwritable-expected-warning {{1 unchecked pointer: Expression in non-writable file}}
```

Comment:
It this expected warning supposed to be here twice?

Reply:
One for the decl (`arr[]`) and for the expression (the initializer) no?

Reply:
OK makes sense. Can you split these onto separate lines using the `@+1` thing?

Reply:
Sure